packages/app/cypress/e2e/runner/logger.cy.ts

```diff
@@ -0,0 +1,43 @@
+import sinon from 'sinon'
```

Comment:
Entire file is not new - just moved from `packages/runner-shared/src/logger.js` and turned into ES6 + TS.